scripts/create-package.js

```diff
@@ -0,0 +1,183 @@
+// Dependencies
+const mustache = require('mustache');
+const argv = require('yargs').argv;
+const fs = require('fs');
+const exec = require('./exec-sync');
+
+// Arguments
+const newPackageName = argv.name;
+const generate = argv.generate === undefined;
+
+// Convert any package names given in dash-case (e.g. my-new-package) to PascalCase (e.g. MyNewPackage)
+// for display purposes in certain template files (e.g. README.md)
+let pascalCasePackage = newPackageName.replace(/-[a-zA-Z]/g, function(m, t) {
+  return newPackageName[t + 1].toUpperCase();
+});
+pascalCasePackage = pascalCasePackage.substring(0, 1).toUpperCase() + pascalCasePackage.substring(1);
+
+// Current date used in CHANGELOG.md
+const today = new Date().toUTCString();
+
+// Paths
+const packagePath = `./packages/${newPackageName}`;
+const templateFolderPath = './scripts/templates/';
+const rushPath = __dirname + '/../rush.json';
+
+// rush.json contents
+let rushJson = require(rushPath);
+
+// Output Files
+const outputFiles = {
```

Comment:
maybe if these are all prefixed like that, you could just prefix it where you need to use this later so you don't have to do it all over the place :)